src/app/shared/services/query/sqlite-query.service.ts

```diff
@@ -14,7 +14,10 @@ export class SqliteQueryService extends BaseQueryService {
   private itemDisplayIdCache: Observable<string>[] = [];
   private spellDisplayIdCache: Observable<string>[] = [];
 
-  constructor(private sqliteService: SqliteService, private configService: ConfigService) {
+  constructor(
+    private sqliteService: SqliteService,
+    private configService: ConfigService,
+  ) {
```

Comment:
use `inject` and get rid of the constructor

Reply:
not sure if we can do this with super() inside the contrusctor and the mother class